cli/code_scanner.py

```diff
@@ -39,12 +39,26 @@
                    "vulnerabilities and violations (supported for SCA scan type only).",
               type=bool,
               required=False)
+@click.option('--package-vulnerabilities',
```

Comment:
we agreed on one option like that represent the sca scan type no?

Reply:
Yes